backEnd/src/themes/themes.controller.ts

```diff
@@ -49,4 +49,32 @@ export class ThemesController {
   create(@Body() body: CreateThemeDto) {
     return this.themesService.create(body);
   }
+
+  @ApiOperation({ summary: 'Update a theme' })
+  @ApiBody({
+    type: CreateThemeDto,
```

Comment:
We usually have another DTO for update action because some of the create fields may not be editable
```suggestion
    type: UpdateThemeDto,
```